CONTRIBUTING.md

```diff
@@ -25,7 +21,7 @@ git clone git@github.com:mapbox/mapbox-gl-js.git
 
 Install node module dependencies
 ```bash
-yarn install
+npm install
```

Comment:
What minimum version of NPM is needed?

Reply:
An appropriate version of npm gets installed whenever a version of node is installed, nobody generally has to think about this.

Reply:
Node 14, which we specify elsewhere, installs npm 6 by default (so does Node 10 and 12). It looks like Node 16 will install npm 7 by default when that becomes available. See https://github.com/nodejs/node/discussions/37689.